plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -166,6 +171,11 @@ public MachineProviderImpl(DockerConnector docker,
         this.doForcePullOnBuild = doForcePullOnBuild;
         this.privilegedMode = privilegedMode;
         this.snapshotUseRegistry = snapshotUseRegistry;
+        if (Strings.isNullOrEmpty(cheUserId)) {
+            this.cheUserId = System.getenv("CHE_USER_ID") == null ? "" : System.getenv("CHE_USER_ID");
```

Comment:
I have tried to rename CHE_USER_ID to CHE_PROPERTY_che_docker_user__id. However, che converts env. variable using cli.sh that is called before configuring CHE_USER_ID (entrypoint.sh) so that CHE_PROPERTY_che_docker_user__id isn't converted properly.

If che server starts with
```
docker run ... eclipse/che-server 
```

cli.sh won't be called.